CMakeLists.txt

```diff
@@ -104,8 +104,8 @@ endif(${WITH_AIO})
 
 option(WITH_FUSE "Fuse is here" ON)
 if(${WITH_FUSE})
-find_package(fuse REQUIRED)
-set(HAVE_LIBFUSE ${FUSE_FOUND})
+find_package(fuse)
```

Comment:
WITH_FUSE defaults to ON, and is optional.  Anything which is optional and defaults to ON should not fail CMake if it's not there.